pages/tutorial/simulations.md

```diff
@@ -147,7 +165,22 @@ More about the environments shipped with the distribution [here][Environments].
 
 #### The `positions` key
 
+The `position` section list the coordination types of the simulation. Actually, only one value is taken into account. Each coordinate implements the interface [Position][Position].
```

Comment:
Instead of listing what is implemented, just link the package, so that if something is added in the code it gets updated here automatically.